internal/grpcrand/grpcrand.go

```diff
@@ -1,3 +1,6 @@
+//go:build !go1.21
+// +build !go1.21
```

Comment:
Would you mind adding a TODO here that says something like:

"TODO: when this file is deleted (after Go 1.20 support is dropped), delete all of grpcrand and call the rand package directly."

?

Reply:
Done

Reply:
Sorry for my absence and thank u @arvindbr8

Reply:
No worries, sorry if you were working on the branch. Since this was a oneliner change, and since you had also allowed edits to your PR by maintainers I went ahead and made the change.